download-sony-blobs.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+ZIP_FILE="SW_binaries_for_Xperia_AOSP_L_MR1_v9.zip"
+URL_BASE="http://dl-developer.sonymobile.com/eula"
+EULA_URL="${URL_BASE}/SW_binaries_for_Xperia_AOSP_Lollipop_MR1_v9_EULA.html"
+
+NOW=$(($(date +"%s") * 1000))
+TOMORROW=$((${NOW} + 1000 * 60 * 60 * 24 * 1))
+
+DIR=download-yukon
+
+if [ ! -x "$(which w3m)" ]; then
```

Comment:
Please move this part just before we call w3m, that's a bug we fixed on other platforms.